kyuubi-server/src/test/scala/org/apache/kyuubi/session/SessionSigningSuite.scala

```diff
@@ -68,16 +70,22 @@ class SessionSigningSuite extends WithKyuubiServer with HiveJDBCTestHelper {
   test("KYUUBI #3839 session user sign - check session user sign") {
     withSessionConf()()() {
       withJdbcStatement() { statement =>
-        val rs1 = statement.executeQuery(s"SET $KYUUBI_SESSION_SIGN_PUBLICKEY")
-        assert(rs1.next())
-        assert(rs1.getString("value").nonEmpty)
+        val value = s"SET ${OPERATION_LANGUAGE.key}=scala"
+        statement.executeQuery(value)
 
-        val rs2 = statement.executeQuery(s"SET $KYUUBI_SESSION_USER_SIGN")
+        val rs1 = statement.executeQuery(
+          "spark.sparkContext.getLocalProperty(\"kyuubi.session.sign.publickey\")")
+        assert(rs1.next())
+        val rs2 = statement.executeQuery(
+          s"spark.sparkContext.getLocalProperty(\"kyuubi.session.user.sign\")")
```

Comment:
KyuubiReservedKeys.KYUUBI_SESSION_USER_SIGN

Reply:
Having trouble in changing it to `s"spark.sparkContext.getLocalProperty(\"${KYUUBI_SESSION_USER_SIGN}\")")`, it comes with a compilation error "value $ is not a member of String".

Reply:
OK, thx for the help. Changing to 
```
          s"""
             |spark.sparkContext.getLocalProperty("$KYUUBI_SESSION_USER_SIGN")
             |""".stripMargin
```